ironfish/src/rpc/routes/wallet/getAccountTransactions.ts

```diff
@@ -86,9 +90,17 @@ routes.register<typeof GetAccountTransactionsRequestSchema, GetAccountTransactio
     let count = 0
     let offset = 0
 
-    const transactions = request.data.sequence
-      ? account.getTransactionsBySequence(request.data.sequence)
-      : account.getTransactionsByTime(undefined, { reverse: request.data.reverse })
+    let transactions
+    if (request.data.startSequence || request.data.endSequence) {
+      transactions = account.getTransactionsBySequenceRange(
+        request.data.startSequence,
+        request.data.endSequence,
+      )
+    } else if (request.data.sequence) {
+      transactions = account.getTransactionsBySequence(request.data.sequence)
```

Comment:
I'm guessing we can just delete this now assuming `getTransactionsBySequenceRange` is inclusive which it looks like it is.

Reply:
It is inclusive, so we can delete `getTransactionsBySequence`